android/app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```

Comment:
@Woynert do we need both permissions, based on [some information](https://www.youtube.com/watch?v=8KJH5jwmOPU) i found, just using the ACCESS_FINE_LOCATION should be OK.